webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue

```diff
@@ -141,8 +141,8 @@ export default {
       }
       CKEDITOR.dtd.$removeEmpty['i'] = false;
 
-      let extraPlugins = 'simpleLink,suggester,widget';
-      let removePlugins = 'image,maximize,resize';
+      let extraPlugins = 'simpleLink,suggester,widget,resize';
```

Comment:
why do you add the resize plugin ?

Reply:
I add the resize plugin to allow vertical enlargement of the textarea.

Reply:
This is not asked in the current task. Can you remove please ?